engine/src/flutter/impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -229,6 +230,10 @@ TextureGLES::~TextureGLES() {
   }
 }
 
+void TextureGLES::Leak() {
+  handle_ = HandleGLES::DeadHandle();
```

Comment:
Does `ReactorGLES::CollectHandle` handle a `DeadHandle`? We should just add an early return in that case since we'll be acquiring locks and such.

Reply:
Added one.